Project2015To2017/Transforms/PropertySimplificationTransformation.cs

```diff
@@ -13,6 +13,12 @@ namespace Project2015To2017.Transforms
 {
 	public sealed class PropertySimplificationTransformation : ITransformation
 	{
+		private static readonly string[] IgnoreProjectNameValues =
```

Comment:
Uhm... What is that?

Reply:
I'll have a look later, a merge issue maybe...

Reply:
It is from #170, matching current code, but still present here and not causing merge conflicts. What?

Reply:
@hvanbakel yeah, that looks like a merge issue. Once PR is cleaned up of this unneeded change — count my approval in effect.

Reply:
If no merge issues occur and no duplicate field definition appears — we might indeed ignore this issue.

Reply:
Yeah I think this just looks weird because of how I merged it (I guess). I did a merge from master.